Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -1021,15 +891,17 @@ bool FindLargestRect(CGPoint point, float rectHeight, CGRect& ret) {
 };
 
 CGRect _CGPathFitRect(CGPathRef pathref, CGRect rect, CGSize maxSize, float padding) {
```

Comment:
Just nuke this function.  This is used elsewhere in UIKit, where this needs to be re-written using public apis (CGPathApply possibly) to get the exclusion path.